src/main/scala/mesosphere/marathon/DebugConf.scala

```diff
@@ -22,7 +22,7 @@ trait DebugConf extends ScallopConf {
     noshort = true,
     prefix = "disable_")
 
-  lazy val deprecatedDebugTracing = opt[Boolean]("enable_tracing", hidden = true)
```

Comment:
This change doesn't modify the default value, it just makes the flag show in the output of `--help`.

`opt[Boolean]` flags are actually [disabled by default](https://github.com/scallop/scallop/wiki/Option-types-and-definitions):

> Special case of plain options are the flag options. They take no arguments, and evaluate to true if option was present on the command line or to false if it was not